packages/selinux-policy/rules.cil

```diff
@@ -26,6 +26,11 @@
 ; Unprivileged components cannot interact with privileged processes.
 (neverallow unprivileged_s privileged_s (processes (interact)))
 
+; Never allow noatsecure to any subject
+(neverallow all_s all_s (process (noatsecure)))
```

Comment:
I'd prefer to scope this to host processes via the existing "relax" rule.

In some ways it doesn't matter since we don't allow container processes to transition to any other domain, but in the hypothetical case that we did, we would not want the host's use of SELinux to enforce secure execution rules inside the container environment.